test/contracts/test.aes

```diff
@@ -84,17 +84,17 @@ contract Identity =
   //let twice(f,x) = f(f(x))
   // let twice(f,x) = f(f(x))
   // let double(x) = x+x
-  // let main(x) = twice((y=>y+y),x)
+  // let main_(x) = twice((y=>y+y),x)
   // let rec map(f,xs) = switch(xs) {
   // | [] => []
   // | (x::ys) => f(x)::map(f,ys)
   // }
   // let id(x) = x
-  // let main(xs) = map(double,xs)
+  // let main_(xs) = map(double,xs)
   entrypoint z(f,x) = x
   function s(n) = (f,x)=>f(n(f,x))
   function add(m,n) = (f,x)=>m(f,n(f,x))
-  entrypoint main(_) =
+  entrypoint main_(_) =
```

Comment:
What's up with `main` is this some new keyword?

Reply:
yes